src/site/content/en/lighthouse-seo/tap-targets/index.md

```diff
@@ -2,61 +2,70 @@
 layout: post
 title: Tap targets are not sized appropriately
 description: |
-  Learn about tap-targets audit.
+  Learn about the "Tap targets are not sized appropriately" Lighthouse audit.
+updated: 2019-08-21
 web_lighthouse:
   - tap-targets
 ---
 
-Tap targets are interactive elements, like buttons or links,
-that users frequently tap.
-Appropriately-sized tap targets make pages more mobile-friendly and accessible.
-In-appropriately sized tap-targets have the opposite effect.
+Tap targets are the areas of a web page that users on touch devices can
+interact with. Buttons, links, and form elements all have tap targets.
 
-A tap target is inappropriately-sized when it's too small, or too close to other tap targets.
-Lighthouse flags pages with inappropriately-sized tap targets:
+Many search engines rank pages based on how mobile-friendly they are. Making
+sure tap targets are big enough and far enough apart from each other makes
+your page more mobile-friendly and accessible.
+
+## How the Lighthouse audit for tap targets fails
+
+Lighthouse flags pages with tap targets that are too small or too close together:
 
 <figure class="w-figure">
   <img class="w-screenshot w-screenshot--filled" src="tap-targets.png" alt="Lighthouse audit showing inappropriately sized tap targets">
-  <figcaption class="w-figcaption">
-    Tap targets are inappropriately sized.
-  </figcaption>
 </figure>
 
-{% Aside 'note' %}
-Google Search started boosting the ranking of mobile-friendly pages
-on mobile search results back in 2015.
-See [Rolling out the mobile-friendly update](https://webmasters.googleblog.com/2015/04/rolling-out-mobile-friendly-update.html).
-{% endAside %}
-
-## What causes this audit to fail
-
-Click the audit to see which tap targets are causing the audit to fail.
-The **Tap Target** column tells you which tap target is inappropriately-sized.
-The **Size** column tells you the size of the target's bounding rectangle, in pixels.
-The **Overlapping Target** column tells you which other tap target is too close.
+Targets that are smaller than 48&nbsp;px by 48&nbsp;px or closer than 8&nbsp;px
```

Comment:
I'm not a fan of special characters like `&nbsp;` ... extra work for little payoff

Reply:
In general, I agree, though preventing units from being orphaned without their values seems important for clarity. It's not something I'm enforcing in reviews (yet), but it's something I try to do in my own stuff.